packages/linea-state-verifier/contracts/LineaSparseProofVerifier.sol

```diff
@@ -40,9 +40,9 @@ contract LineaSparseProofVerifier is IEVMVerifier {
         // Check that the L2 block number used is a recent one and is part of the range accepted
         uint256 currentL2BlockNumber = IRollup(_rollup).currentL2BlockNumber();
         require(
-            (currentL2BlockNumber <= l2BlockRangeAccepted &&
+            (currentL2BlockNumber <= acceptedL2BlockRangeLength &&
                 blockNo <= currentL2BlockNumber) ||
-                blockNo >= currentL2BlockNumber - l2BlockRangeAccepted,
+                blockNo >= currentL2BlockNumber - acceptedL2BlockRangeLength,
```

Comment:
I would bracket off the `||` to be explicit here for where the `&&` starts and end